github/repos_commits.go

```diff
@@ -222,9 +223,11 @@ func (s *RepositoriesService) GetCommitSHA1(ctx context.Context, owner, repo, re
 
 // CompareCommits compares a range of commits with each other.
 //
+// The url string u is escaped before making the request to api
```

Comment:
```suggestion
// The url query string is escaped before making the request.
```

Reply:
Hi, acutally I store the escaped value for head and base in `h` and `b` and use the variables in the test instead of using literal values.